templates/course-results/lessons.php

```diff
@@ -82,7 +82,7 @@
 						}
```

Comment:
Any time a template is changed, the version at the top needs to be updated. This will likely go in version 3.2.0.